test/libponyc/verify.cc

```diff
@@ -492,3 +492,11 @@ TEST_F(VerifyTest, FFICallPartialWithNonPartialDeclaration)
 
   TEST_ERRORS_1(src, "call is partial but the declaration is not");
 }
+
+TEST_F(VerifyTest, LambdaTypeGenericAliased)
+{
+  const char *good =
+    "type Foo[T] is { (T): T }";
```

Comment:
To be consistent with the lambda type style used in other tests, could you remove the "extra" spaces inside the curly braces.  That is, it would be consistent if it looked like: `{(T): T}`.